sustc-api/src/main/java/io/sustc/service/VideoService.java

```diff
@@ -120,7 +120,6 @@ public interface VideoService {
      *   <li>{@code auth} is invalid, as stated in {@link io.sustc.service.UserService#deleteAccount(AuthInfo, long)}</li>
      *   <li>{@code keywords} is invalid (null or empty)</li>
      *   <li>{@code pageSize} and/or {@code pageNum} is invalid (any of them <= 0)</li>
-     *   <li>the video is not visible to current user</li>
```

Comment:
why remove it? we can instead add a clarification about whether a video is visible or not

Reply:
No, this line is incorrect since one may almost always have searched for invisible videos, and these videos should simply disappear from the resulting list rather than returning null. The line 110 `Unreviewed or unpublished videos are only visible to superusers or the video owner.` is enough.